app/scripts/controllers/preferences.js

```diff
@@ -534,6 +537,12 @@ export default class PreferencesController {
     return this.store.getState().disabledRpcMethodPreferences;
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IN(snaps)
```

Comment:
I don't think we should fence this with the snap feature flag. That would mean that this goes into stable when we ship snaps to stable. It should either be under the Flask feature flag or a separate one I think 🤔